### PR TITLE
favorite package feature flag

### PR DESCRIPTION
**Pull request by FarhanAliRaza**
fixes https://github.com/djangopackages/djangopackages/issues/1227

### CODE REVIEW

**Review by saadmk11 — COMMENTED** [review]
A test is failing probably because `django-waffle` `templatetag` performs a database query which breaks the `assertNumQueries` assertion on that test.

Should we set `WAFFLE_CREATE_MISSING_FLAGS = True` in Django settings?

**Comment by jefftriplett** [comment]
> Should we set `WAFFLE_CREATE_MISSING_FLAGS = True` in Django settings?

Yes plz.

File: package/tests/test_views.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def test_package_list_view(self):
 
     def test_package_detail_view(self):
         url = reverse("package", kwargs={"slug": "testability"})
-        with self.assertNumQueries(16):
+        with self.assertNumQueries(21):  # 16 expected 21 because of WAFFLE FLAG
```
Comment (jefftriplett) [inline package/tests/test_views.py:105]: This is a strangely high number. 🤔

**Review by jefftriplett — APPROVED** [review]
Looks good!

**Pull request merged**